org.team_hermes.WireShare.yaml

```diff
@@ -0,0 +1,35 @@
+id: org.team_hermes.WireShare
+branch: 6.0
+runtime: org.freedesktop.Platform
+runtime-version: 19.08
+sdk: org.freedesktop.Sdk
+command: /app/WireShare.sh
+modules:
```

Comment:
I think you need  org.freedesktop.Sdk.Extension.openjdk11 extension.

Reply:
I'm vendoring my own, somewhat lobotomised OpenJDK; we have found that it works quite a bit faster than the Oracle-supplied OpenJDK.